docs/index.bs

```diff
@@ -2857,8 +2876,9 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
           1. If |shouldSoftUpdate| is true, then [=in parallel=] run the [=Soft Update=] algorithm with |registration|.
           1. Return null.
       1. If |activeWorker|'s <a>state</a> is *activating*, wait for |activeWorker|'s <a>state</a> to become *activated*.
-      1. Invoke <a>Run Service Worker</a> algorithm with |activeWorker| as the argument.
-      1. <a>Queue a task</a> |task| to run the following substeps:
+      1. Run the [=Run Service Worker=] algorithm with |activeWorker|.
```

Comment:
Do you prefer "Run the" to "Invoke the"? Happy to go with "Run the".

Reply:
Ah, meant to bring this up earlier. There's some inconsistency around "Invoke" vs "Run" and also "with xx, yy, and zz" vs "passing xx, yy, and zz as arguments" and other variations. I planned to just always go with the shortest which is "Run the AA algorithm with xx, yy, and zz." But not sure if there is a style guideline here.

I guess with Run in the name of algorithm, it looks a bit awkward.

Reply:
It probably doesn't matter much. In other specs I avoid it by making the link text an action, like "Let foo be the result of getting a service worker object…". But I guess we should keep stuff in the algorithm section more like a function.

"Run" or "invoke" seems fine here. No point trying to solve the inconsistencies in this PR.